devicetypes/smartthings/ozom-smart-siren.src/ozom-smart-siren.groovy

```diff
@@ -0,0 +1,165 @@
+	}
+
+	tiles {
+		standardTile("alarm", "device.alarm", width: 2, height: 2) {
+			state "off", label:'off', action:'alarm.siren', icon:"st.secondary.siren", backgroundColor:"#ffffff"
+			state "siren", label:'siren!', action:'alarm.off', icon:"st.secondary.siren", backgroundColor:"#e86d13"
+		}
+
+		main "alarm"
+		details(["alarm"])
+	}
+}
+
+private getDEFAULT_MAX_DURATION() { 0x00B4 }
+private getDEFAULT_DURATION() { 0xFFFE }
+
+private getIAS_WD_CLUSTER() { 0x0502 }
+
+private getATTRIBUTE_IAS_WD_MAXDURATION() { 0x0000 }
+private getATTRIBUTE_IAS_ZONE_STATUS() { 0x0002 }
+
+private getCOMMAND_IAS_WD_START_WARNING() { 0x00 }
+private getCOMMAND_DEFAULT_RESPONSE() { 0x0B }
+
+def turnOffAlarmTile(){
+	sendEvent(name: "alarm", value: "off")
+}
+
+def turnOnAlarmTile(){
+	sendEvent(name: "alarm", value: "siren")
+}
+
+def installed() {
+	sendCheckIntervalEvent()
+	state.maxDuration = DEFAULT_MAX_DURATION
+	turnOffAlarmTile()	
+}
+
+def parse(String description) {
+	log.debug "Parsing '${description}'"
+
+	Map map = zigbee.getEvent(description)
+	if (!map) {
+		if (description?.startsWith('zone status')) {
+			map = parseIasMessage(zigbee.parseZoneStatus(description))
+		} else if (description?.startsWith('zone report')){
+			map = parseIasMessage(parseZoneReport(description))
+		} else if (description?.startsWith('enroll request')) {
+			List cmds = zigbee.enrollResponse()
+			log.debug "enroll response: ${cmds}"
+			return cmds
+		} else {
+			Map descMap = zigbee.parseDescriptionAsMap(description)
+			if (descMap?.clusterInt == IAS_WD_CLUSTER) {
+				def data = descMap.data
+
+				Integer parsedAttribute = descMap.attrInt
+				Integer command = Integer.parseInt(descMap.command, 16)
+				if (parsedAttribute == ATTRIBUTE_IAS_WD_MAXDURATION && descMap?.value) {
+					state.maxDuration = Integer.parseInt(descMap.value, 16)
+				} else if (command == COMMAND_DEFAULT_RESPONSE) {
+					Boolean isSuccess = Integer.parseInt(data[-1], 16) == 0
+					Integer receivedCommand = Integer.parseInt(data[-2], 16)
+					if (receivedCommand == COMMAND_IAS_WD_START_WARNING && isSuccess){
+						if(state.alarmOn){
+							turnOnAlarmTile()
+							runIn(state.lastDuration, turnOffAlarmTile)
+						} else {
+							turnOffAlarmTile()
+						}
+					}
+				}
+			}
+		}
+	}
+	log.debug "Parse returned $map"
+	def results = map ? createEvent(map) : null
+	log.debug "parse results: " + results
+	return results
+}
+
+private Map parseIasMessage(ZoneStatus zoneStatus) {
+	Map resultMap = [name: 'DeviceWatch-DeviceStatus']
+
+	if(zoneStatus.isAcSet()) {
+		turnOffAlarmTile()
+		resultMap << [value: 'offline', description: 'Lost power']
+	} else {
+		resultMap << [value: 'online', description: 'Power restored']
+	}
+	return resultMap
+}
+
+private ZoneStatus parseZoneReport(String description) {
```

Comment:
You could just use `zigbee.parseZoneStatus()` again here and then get the `value` attribute.